### PR TITLE
Added contribution and PR guidelines

### PR DESCRIPTION
**Pull request by soumeh01**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        1. With this config...
+        1. Run '...'
+        1. See error...
+    validations:
+      required: false
+  - type: checkboxes
```
Comment (jkrech) [inline .github/ISSUE_TEMPLATE/bug.yaml:36]: I would propose that the issues reported in this repo should only be about cbuild2cmake.
Do we really need these check-boxes for all other tools in the CMSIS-Toolbox?

Comment (soumeh01) [reply .github/ISSUE_TEMPLATE/bug.yaml:36]: [Addressed review comments](https://github.com/Open-CMSIS-Pack/cbuild2cmake/pull/224/commits/71d9d899507ed10f50435ded3a371af5f7779778)

File: .github/ISSUE_TEMPLATE/blocker.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        - label: packchk
+          required: false
+        - label: svdconv
+          required: false
+  - type: checkboxes
+    attributes:
+      label: Affected operating systems
+      description: On which host os platforms does the bug occur?
+      options:
+        - label: Windows amd64
```
[the remark this answers is not in the document]
Comment (soumeh01) [reply .github/ISSUE_TEMPLATE/blocker.yaml:64]: [Addressed review comments](https://github.com/Open-CMSIS-Pack/cbuild2cmake/pull/224/commits/71d9d899507ed10f50435ded3a371af5f7779778)

**Review by jkrech — APPROVED** [review]
LGTM

**Pull request merged**